### PR TITLE
Fix pathPrefix in source/config.go

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
The files are mounted /host-etc/os-release /host-sys/... /host-boot/... but the pathPrefix  in Makefile and Dockerfile was set to /

Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,9 +5,10 @@ WORKDIR /go/node-feature-discovery
 # Do actual build
 COPY . /go/node-feature-discovery
 
-RUN go install \
-  -ldflags "-X sigs.k8s.io/node-feature-discovery/pkg/version.version=v0.6.0" \
-  ./cmd/*
+ARG VERSION=v0.6.0
+ARG HOSTMOUNT_PREFIX=/host-
+
+RUN make install VERSION=$VERSION HOSTMOUNT_PREFIX=$HOSTMOUNT_PREFIX
```
Comment (sjug) [inline Dockerfile:11]: Can we curly brace the vars here?

Comment (ArangoGutierrez) [reply Dockerfile]: fd0c4c1

**Pull request merged**